<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAppMainWindow</class>
 <widget class="QMainWindow" name="QAppMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFEditor v0.5.0</string>
  </property>
  <widget class="QTabWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="action_Close_All"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Save_All"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="action_Print_Preview"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Select_All"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
    <addaction name="action_Goto_Line"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="action_Open_Project"/>
    <addaction name="action_Save_Project"/>
    <addaction name="action_Close_Project"/>
    <addaction name="separator"/>
    <addaction name="action_Project_Properties"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Check_Project"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_Full_Screen"/>
    <addaction name="action_Hide_Everything"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_ColdFusion_Help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Project"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="file_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Save_All"/>
   <addaction name="separator"/>
   <addaction name="action_Print"/>
   <addaction name="action_Print_Preview"/>
  </widget>
  <widget class="QToolBar" name="edit_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="separator"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Find"/>
  </widget>
  <widget class="QDockWidget" name="m_ProjectDock">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>98</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Project Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QAdvancedTreeWidget" name="m_ProjectTree">
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_StructureDock">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>98</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Structure</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QAdvancedTreeWidget" name="m_StructureTree">
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QToolBar" name="project_toolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open_Project"/>
   <addaction name="action_Save_Project"/>
   <addaction name="action_Close_Project"/>
   <addaction name="separator"/>
   <addaction name="action_Project_Properties"/>
  </widget>
  <widget class="QToolBar" name="view_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Full_Screen"/>
   <addaction name="action_Hide_Everything"/>
  </widget>
  <widget class="QDockWidget" name="m_BrowserDock">
   <property name="windowTitle">
    <string>Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="m_Browser2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWebEngineView" name="m_Browser">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_New">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/new.png</normaloff>:/CFEditor/Commands/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open fiile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close current file</string>
   </property>
  </action>
  <action name="action_Close_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="toolTip">
    <string>Close all open files</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/save.png</normaloff>:/CFEditor/Commands/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/save_as.png</normaloff>:/CFEditor/Commands/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save current file as</string>
   </property>
  </action>
  <action name="action_Save_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/save_all.png</normaloff>:/CFEditor/Commands/save_all.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="toolTip">
    <string>Save all modified files</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/print.png</normaloff>:/CFEditor/Commands/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="toolTip">
    <string>Prints current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Print_Preview">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/print_preview.png</normaloff>:/CFEditor/Commands/print_preview.png</iconset>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
   <property name="toolTip">
    <string>Previews current file</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the IDE</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/cut.png</normaloff>:/CFEditor/Commands/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/paste.png</normaloff>:/CFEditor/Commands/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Select_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/find.png</normaloff>:/CFEditor/Commands/find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Goto_Line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Goto &amp;Line</string>
   </property>
   <property name="toolTip">
    <string>Goto Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_Open_Project">
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open Project</string>
   </property>
  </action>
  <action name="action_Save_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
   <property name="toolTip">
    <string>Save Project</string>
   </property>
  </action>
  <action name="action_Close_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
   <property name="toolTip">
    <string>Close Project</string>
   </property>
  </action>
  <action name="action_Project_Properties">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/settings.png</normaloff>:/CFEditor/Commands/settings.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;roperties</string>
   </property>
   <property name="toolTip">
    <string>Project Properties</string>
   </property>
  </action>
  <action name="action_Check_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/validate.png</normaloff>:/CFEditor/Commands/validate.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Check Project</string>
   </property>
   <property name="toolTip">
    <string>Check Project</string>
   </property>
  </action>
  <action name="action_Full_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/fullscreen.png</normaloff>:/CFEditor/Commands/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="action_ColdFusion_Help">
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/help.png</normaloff>:/CFEditor/Commands/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ColdFusion Help</string>
   </property>
   <property name="toolTip">
    <string>ColdFusion Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="cfeditor.qrc">
     <normaloff>:/CFEditor/Commands/about.png</normaloff>:/CFEditor/Commands/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About this program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="action_Hide_Everything">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hide Everything</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
  <customwidget>
   <class>QAdvancedTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">qadvancedtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cfeditor.qrc"/>
 </resources>
 <connections/>
</ui>
